storage/plugins/archives/ArchivesPlugin.php

```
@@ -5224,6 +5271,241 @@ public function iiifCollectionAction(
+        ServerRequestInterface $request,
+        ResponseInterface $response
+    ): ResponseInterface {
+        $rootUnits = [];
+        $result = $this->db->query(
+            "SELECT id, level, constructed_title, formal_title
+               FROM archival_units
+              WHERE parent_id IS NULL AND deleted_at IS NULL AND level = 'fonds'
+              ORDER BY reference_code"
+        );
+        if ($result instanceof \mysqli_result) {
+            while ($r = $result->fetch_assoc()) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)